db/migrate/20151018090641_create_persistent_volume_claim.rb

```diff
@@ -0,0 +1,17 @@
+class CreatePersistentVolumeClaim < ActiveRecord::Migration
+  def change
+    create_table :persistent_volume_claims do |t|
+      t.belongs_to :ems, :type => :bigint
+      t.belongs_to :persistent_volume, :type => :bigint
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :ems_created_on
+      t.timestamp  :created_on
```

Comment:
My [comment from the other thread](https://github.com/ManageIQ/manageiq/pull/5926#discussion-diff-60133969) was lost:

Prefer t.timestamps over your own created_on column. Rails uses this column automatically and changes the value, so if the intention was to use it for native created_on dates from the provider, you have to use a different column name.